net-dns/dnscap/Manifest

```diff
@@ -1 +1 @@
-DIST dnscap-20130814.tar.gz 99615 BLAKE2B e663349e4acc20b88f7aee14676736534746c54072a4bf5d9ab9aa9935781a39f4a94d0d718108aba6b4035390214d76caf62f9cd71b364507ec55c7e93fd07d SHA512 1969d8cc47c6206369a02b29064dbb7f663a2187203ab1f49d862082bfb6e44c9fccf90fb63d65a523c7a5cf4d24815a74b9dd18d81267b4447252d7a04f33dd
```

Comment:
1. Add new version and remove old in separate commits.
2. Unless the old version is broken beyond usability or otherwise completely useless, please keep it for a few days in case upgrade went horribly wrong for users.

Reply:
Ok, now readded the old ebuild